### PR TITLE
Add missing -ManagementPointNetworkType parameter to New-Cluster documentation

### PR DESCRIPTION
**Pull request by peterschen**
# PR Summary

Add missing `-ManagementPointNetworkType` parameter to the `New-Cluster` cmdlet documentation, which is available [since Windows Server 2019](https://learn.microsoft.com/en-us/powershell/module/failoverclusters/new-cluster?view=windowsserver2019-ps#-managementpointnetworktype).

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Comment by peterschen** [comment]
@microsoft-github-policy-service agree

File: docset/winserver2022-ps/failoverclusters/New-Cluster.md [hunk 1]
```diff
@@ -167,6 +167,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ManagementPointNetworkType
+Specifies the network configuration used to determine IP address settings.
+
+The acceptable values for this parameter are:
+
+- Automatic: Automatically detects the appropriate setting. If SQL Server is running in Azure, uses `Distributed`. If SQL Server is running on-premises, uses `Singleton`. (Default setting)
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/failoverclusters/New-Cluster.md:175]:
```suggestion
- Automatic: Automatically detects the appropriate setting. If SQL Server is running in Azure, it uses `Distributed`. If SQL Server is running on-premises, it uses `Singleton`. (Default setting)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @peterschen, kindly commit the minor suggested change. Many thanks.

File: docset/winserver2022-ps/failoverclusters/New-Cluster.md (hunk 1 above)
Comment (peterschen) [reply docset/winserver2022-ps/failoverclusters/New-Cluster.md]: Fixed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by alexjmoore** [comment]
Hi @JohanFreelancer9 - just wondering if there was an update to this PR?

**Comment by JohanFreelancer9** [comment]
> Hi @JohanFreelancer9 - just wondering if there was an update to this PR?

@scanum, can you please help to get this PR reviewed and merged? Thanks.

**Review by get-itips — COMMENTED** [review]
Can you please make the same additions for the Windows Server 2019 version of the article? Thank you

**Comment by alexjmoore** [comment]
Hi @get-itips - in fact these additions are already in the 2019 docs, but were omitted from the 2022, which this PR aims to correct.

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tseward hope all is well. Could you please help us review? Thanks!

**Comment by alexjmoore** [comment]
Any update on this one?

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by alexjmoore** [comment]
Any update?

**Comment by scanum** [comment]
@tiburd
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
close > reopen

**Pull request merged**